set.mm

```diff
@@ -622795,6 +622865,18 @@ fixed reference functional determined by this vector (corresponding to
       AEUPUHUIPQUJ $.
   $}
 
+  ${
+    $d w x y z $.  $d ph w y z $.
+    $( ~ ax-sep without ~ ax-9 to ~ ax-13 .  (Contributed by SN,
+       25-Sep-2023.) $)
+    sn-axsep $p |- E. y A. x ( x e. y <-> ( x e. z /\ ph ) ) $=
```

Comment:
Better than ~bj-axsep ! I'll look at it on the website, and we may move it to Main !